CHANGELOG/CHANGELOG-2.5.md

```diff
@@ -0,0 +1,80 @@
+- Refine Traceflow to correctly handle inter-Node Pod-to-Pod traffic across all traffic modes. ([#7481](https://github.com/antrea-io/antrea/pull/7481), [@hongliangl])
+- Fix ACNP applied to NodePort failing to reject traffic in `noEncap`/`hybrid` mode. ([#7265](https://github.com/antrea-io/antrea/pull/7265), [@hongliangl])
+- Fix a type assertion panic in `GetFlowTableID` function, which affected the `/ovsflows` HTTP handler. ([#7515](https://github.com/antrea-io/antrea/pull/7515), [@antoninbas])
+- Handle missing Pod IP and Pod IP changes in NodePortLocal to prevent incorrect datapath rules and clear NPL annotations. ([#7512](https://github.com/antrea-io/antrea/pull/7512), [@antoninbas])
+- Exclude Egress VLAN sub-interfaces (`antrea-ext.VLAN`) from NodePort addresses for consistency. ([#7519](https://github.com/antrea-io/antrea/pull/7519), [@antoninbas])
+- Improve `initK8sNodeLocalConfig` in Agent initialization by separating Node and PodCIDR polling for better logging, and increase the timeout to 60s. ([#7473](https://github.com/antrea-io/antrea/pull/7473), [@antoninbas])
+- Fix panic in monitor controller caused by unexpected delete event type. ([#7568](https://github.com/antrea-io/antrea/pull/7568), [@luolanzone])
+- Clean up stale secondary IPs in IPPool when Node restarts with invalid OVSDB. ([#7511](https://github.com/antrea-io/antrea/pull/7511), [@luolanzone])
+- Improve stale IP recycling in AntreaIPAM controller. ([#7538](https://github.com/antrea-io/antrea/pull/7538) [#7571](https://github.com/antrea-io/antrea/pull/7571), [@luolanzone])
+- Handle Traceflow external destination IP correctly in NoEncap mode to fix timeout issue. ([#7266](https://github.com/antrea-io/antrea/pull/7266), [@gran-vmv])
+- Unify validation logic for IPPool and ExternalIPPool for more consistent checks and failures. ([#7319](https://github.com/antrea-io/antrea/pull/7319), [@wenqiq])
+- Add validation to ensure IP range start is not greater than end in IPPool. ([#7308](https://github.com/antrea-io/antrea/pull/7308), [@wenqiq])
+- Improve SR-IOV device assignment to ensure it's idempotent. ([#7322](https://github.com/antrea-io/antrea/pull/7322), [@luolanzone])
+- Improve secondary interface reconciliation and fix a nil pointer exception when both SR-IOV and VLAN interfaces are enabled in Antrea SecondaryNetwork. ([#7286](https://github.com/antrea-io/antrea/pull/7286), [@jianjuns])
+- Add missing Run calls for nodeStore / serviceStore to start the garbage collection routines and fix a memory leak for FlowAggregator. ([#7343](https://github.com/antrea-io/antrea/pull/7343), [@antoninbas])
+- Remove trailing whitespace from default manifests to fix antrea-config ConfigMap formatting issues. ([#7311](https://github.com/antrea-io/antrea/pull/7311), [@antoninbas])
+
+[@Atish-iaf]: https://github.com/Atish-iaf
+[@Dyanngg]: https://github.com/Dyanngg
```

Comment:
you forgot to add Yang's PR: https://github.com/antrea-io/antrea/pull/7496
probably belongs towards the top of the `Fixed` section